opl/generators/notifications_ingress.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import opl.generators.generic
+
+
+class NotificationsMessagesGenerator(opl.generators.generic.GenericGenerator):
+    def __init__(self, count=1, template="notifications_ingres_template.json.j2"):
+        super().__init__(count=count, template=template, dump_message=False)
+
+    def _data(self):
+        account = self._get_account()
```

Comment:
Given you only use that variable once, might be nicer to just put `self._get_account()` below. But again, this it totally not a blocker.